depthAtSurface: 10.1591644287
inTopside: true
inStateMachine: false
inNavigation: false
inVisionTracking: true
